src/components/carousel/NumbersDotsPagination.tsx

```diff
@@ -0,0 +1,86 @@
+import React, { FC, useCallback } from 'react';
+import { EmblaCarouselType } from 'embla-carousel-react';
+
+import { useMultiStyleConfig } from '@chakra-ui/react';
+
+import Flex from '../flex';
+import Box from '../box';
+import NumbersPaginationButton from './NumbersPaginationButton';
+import DotsPaginationButton from './DotsPaginationButton';
+
+type PaginationType = 'dot' | 'number';
+
+interface Props {
+  currentSlideIndex: number;
+  numberOfSlides: number;
+  mainCarousel?: EmblaCarouselType;
+  paginationType: PaginationType;
+}
+
+const NumbersDotsPagination: FC<Props> = ({
```

Comment:
I wouldn't have them together anymore since it's not clickable. You would create an unnecessary onClick handler for the dots